test/e2e/scheduler_predicates.go

```diff
@@ -297,7 +297,7 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 		framework.ExpectNoError(err)
 		for _, pod := range pods.Items {
 			_, found := nodeToCapacityMap[pod.Spec.NodeName]
-			if found && pod.Status.Phase == api.PodRunning {
+			if found && pod.Status.Phase != api.PodSucceeded && pod.Status.Phase != api.PodFailed {
```

Comment:
How does it fix the problem? In what state the pod was so that it helps? Was it pending?